pkg/cloudprovider/aws/testdata/br_userdata_input.golden

```diff
@@ -0,0 +1,4 @@
+[settings.kubernetes.eviction-hard]
```

Comment:
thoughts on in-lining these in the suite_tests so that you don't have to flip back-and-forth?

Reply:
I don't mind. I like having the input as dedicated files too because it's easier to edit and maintain rather than large multiline strings but I can move it into the tests